Export PrerenderIO object

glasser opened the pull request:
This allows you to (eg) mutate PrerenderIO.crawlerUserAgents.

glasser committed:
Export PrerenderIO object

This allows you to (eg) mutate PrerenderIO.crawlerUserAgents.

benjamn reviewed (approved):
This works correctly, though I have some other minor feedback if we're open to using the `modules` package here.

Commit:
Use modules package to get support for CommonJS require/exports.

Commit:
Use require/exports instead of Npm.require and "global" assignment.

Commit:
Use original prerenderio identifier name to reduce diff size.

The name of this local variable isn't that important, but the only reason I changed it in the previous commit was to reduce the size of the diff, so I'd like to get it right.

Commit:
Use api.mainModule instead of api.addFiles.

glasser commented:
Thanks!

glasser committed:
Upgrade the underlying package while we're at it
